examples/ansi.luau

```diff
@@ -5,4 +5,7 @@ print(ansi.bgRed("This is red background!"))
 print(ansi.bold("This is bold text!"))
 print(ansi.underline("This is underlined text!"))
 print(ansi.dim(ansi.italic("This is dim italic text!")))
+
 print(ansi.red(`red({ansi.blue(`blue({ansi.green("green")})`)})`))
+
+print(ansi.combine(ansi.red, ansi.bold)("This is bold red text!"))
```

Comment:
isn't this more verbose than `ansi.red(ansi.bold("This is bold red text!"))`

Reply:
It is, this is an artifact of using examples as testing.